pkg/provider/pods.go

```diff
@@ -265,6 +265,61 @@ func getCNCFNetworksNamesFromPodAnnotation(networksAnnotation string) []string {
 	return networkNames
 }
 
+// isNetworkAttachmentDefinitionSRIOVConfigMTUSet is a helper function to check whether a CNI config
```

Comment:
Remove this commented code.

Reply:
Actually this was on purpose.  This is also done on other functions in this `pods.go` file to show examples of the JSON that is being parsed.